DevEdu.API/DevEdu.Business.Tests/TestDataHelpers/StudentAnswerOnTaskData.cs

```diff
@@ -0,0 +1,571 @@
+﻿using System;
+using System.Collections.Generic;
+using DevEdu.DAL.Models;
+
+namespace DevEdu.Business.Tests
+{
+    public class StudentAnswerOnTaskData
+    {
+        public const int ExpectedStudentAnswerId = 24;
+        public const int UserId = 1;
+        public const int AsnwerId = 1;
+        public const int TaskId = 1;
+        public const int CheckedStatus = 1;
+        public const int AcceptedStatus = 2;
+        public const int ReturnedStatus = 3;
```

Comment:
it's a bad practice to have such values as consts in static class
it's better to declare them inside test methods